mountpoint-s3-fs/src/memory/pool.rs

```diff
@@ -100,13 +103,15 @@ impl PagedPool {
         match self.inner.get_pool_for_size(size) {
             Some(pool) => {
                 let buffer_ptr = pool.reserve(kind);
-                metrics::histogram!("pool.reserve", "type" => "primary", "kind" => kind.as_str()).record(size as f64);
+                metrics::histogram!("pool.reserved_bytes", "type" => "primary", "kind" => kind.as_str())
+                    .record(size as f64);
                 let slack = pool.buffer_size() - size;
-                metrics::histogram!("pool.slack", "kind" => kind.as_str()).record(slack as f64);
+                metrics::histogram!("pool.slack_bytes", "kind" => kind.as_str()).record(slack as f64);
```

Comment:
Also, do you think it's worth including the pool size here? That would help us deciding if we need to add new sizes